docs/docsite/rst/getting_started_ee/build_execution_environment.rst

```diff
@@ -0,0 +1,61 @@
+.. _building_execution_environments:
+
+######################
+Building your first EE
+######################
+
+We are going to build an EE that represents an Ansible control node containing standard packages like ``ansible-core`` and Python in addition to
+an Ansible collection (``community.postgresql``) and its dependency (the ``psycopg2-binary`` Python connector).
+
+To build your first ee, do the following:
+
+1. Create a project folder on your filesystem.
+
+.. code-block:: bash
+
+  mkdir my_first_ee && cd my_first_ee
+
+2. Create a ``execution-environment.yml`` file that specifies dependencies to include in the image.
+
+.. code-block:: yaml
+
+  cat > execution-environment.yml<<EOF
+  ---
+  version: 3
+
+  dependencies:
+    galaxy:
+      collections:
+      - name: community.postgresql
+  EOF
+
+.. note::
+
+  The ``psycopg2-binary`` Python package is included in the ``requirements.txt`` file for the collection.
+  For collections that do not include ``requirements.txt`` files, you need to specify Python dependencies explicitly.
```

Comment:
might be a  good place to add 'See the Builder documentation  for details.'  And make that an external link

Reply:
Yeah, i think it's a right place and a good exception to include the link